pkg/agent/agent.go

```diff
@@ -730,6 +730,15 @@ func (i *Initializer) setupGatewayInterface() error {
 	if err := i.setInterfaceMTU(i.hostGateway, i.networkConfig.InterfaceMTU); err != nil {
 		return err
 	}
+	// Fix #5451
+	// Set arp_announce to 1 on Linux platform to make the ARP requests from host to gateway
+	// interface always use gateway IP as source IP. These ARP requests without gateway IP will
```

Comment:
"the ARP requests from host to gateway interface" doesn't sound correct.

Set arp_announce to 1 on Linux platform to make the ARP requests sent on the gateway interface always use the gateway IP as the source IP, otherwise the ARP requests would be dropped by ARP SpoofGuard flow.

Reply:
Updated code comment, commit message and PR description.